docs/build_version_doc/build_doc.sh

```diff
@@ -136,9 +136,13 @@ fi
 if [ $latest_tag != ${tag_list[0]} ]
 then
     total=${#tag_list[*]}
-    for (( i=0; i<=$(( $total -1 )); i++ ))
+    for (( i=0; i<=$(( $total - 1 )); i++ ))
```

Comment:
That's a really strange way to do a loop condition. Usually you would do i < $total, right?

Reply:
Yeah, legacy :D